### PR TITLE
Add support for spinal rootlets segmentation

### PR DESCRIPTION
**Pull request by KaterinaKrejci231054**
This draft PR adds an option to correct rootlets segmentation.

Resolves: [#56](https://github.com/spinalcordtoolbox/manual-correction/issues/56)

### CODE REVIEW

File: manual_correction.py [hunk 1]
```diff
@@ -649,6 +658,11 @@ def generate_qc(fname, fname_label, task, fname_qc, subject, config_file, qc_les
             archive_qc(fname_qc, config_file)
         else:
             print("WARNING: SC segmentation file not found: {}. QC report will not be generated.".format(fname_seg))
+
+    # for rootlets segmentation do not make QC
```
Comment (valosekj) [inline manual_correction.py:662]:
```suggestion
    # Skip QC for the spinal rootlets segmentation as `sct_qc` does not support it
    # Context: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4166#issuecomment-1654175610
```

**Comment by valosekj** [comment]
You can also add rootlets into this list: https://github.com/spinalcordtoolbox/manual-correction/blob/main/README.md?plain=1#L5-L12

**Comment by valosekj** [comment]
I added some unit tests for recently implemented `get_orientation` and `change_orientation` functions (https://github.com/spinalcordtoolbox/manual-correction/pull/100) in https://github.com/spinalcordtoolbox/manual-correction/pull/98/commits/cde09167bfab0af745358a91e6076e8f8cfbe493. 
The tests worked fine locally. But when triggered using the GitHub action, the tests failed on missing `sct_image` (which is used by `get_orientation` and `change_orientation`, e.g., [here](https://github.com/spinalcordtoolbox/manual-correction/blob/0d0b61bf6a461e1d81c93e1fa7d887f1268fb829/utils.py#L358)).

image

`sct_image` is missing because we do not currently install SCT for [our CI](https://github.com/spinalcordtoolbox/manual-correction/blob/main/.github/workflows/ci.yml). Not sure if we want to install SCT for our CI as it would increase the CI time a lot...

**Review by valosekj — APPROVED** [review]
Great work, @KaterinaKrejci231054! I tested the PR using the following command, and it worked perfectly! 

```console
python manual_correction.py -path-img ~/data/results/philadelphia-pediatric/philadelphia-pediatric_2024-06-26/data_processed -path-label ~/data/results/philadelphia-pediatric/philadelphia-pediatric_2024-06-26/derivatives/labels -change-orient RPI -path-out ~/data/results/philadelphia-pediatric/philadelphia-pediatric_2024-06-26/derivatives_corr/labels -config ~/data/results/philadelphia-pediatric/philadelphia-pediatric_2024-06-26/three_subjects_to_correct.json -fsleyes-cm jet_matplotlib
```

I'm pre-approving the PR. 
Can you please address [this comment](https://github.com/spinalcordtoolbox/manual-correction/pull/98#issuecomment-2195386900)?
I'll address the [failing CI](https://github.com/spinalcordtoolbox/manual-correction/pull/98#issuecomment-2197796281). Then, we can merge!

**Comment by valosekj** [comment]
Hey, @joshuacwnewton! I added SCT installation into the manual_correction CI in https://github.com/spinalcordtoolbox/manual-correction/pull/98/commits/d58dfeefbd4b51c5393848898ec0fc18062cb9c4 and https://github.com/spinalcordtoolbox/manual-correction/pull/98/commits/56b375fd8ad903ff584d08b8aa4e034fe952afd2, but CI [still struggles](https://github.com/spinalcordtoolbox/manual-correction/actions/runs/9962923549/job/27527777512?pr=98) to locate `sct_image`:

```bash
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/subprocess.py:1720: FileNotFoundError
=========================== short test summary info ============================
FAILED tests/test_utils.py::test_get_orientation - FileNotFoundError: [Errno 2] No such file or directory: 'sct_image'
FAILED tests/test_utils.py::test_change_orientation - FileNotFoundError: [Errno 2] No such file or directory: 'sct_image'
========================= 2 failed, 9 passed in 0.40s ==========================
``` 

Do you have any ideas on how to solve this?

**Comment by joshuacwnewton** [comment]
Thanks for the ping! I have a bit of a backlog to get through since coming back from the conference I was attending, but I'm very happy to take a look at this soon. :)

File: .github/workflows/ci.yml
Comment (valosekj) [inline .github/workflows/ci.yml:1]: oh man, this is a very cool hack! thank you!

**Comment by valosekj** [comment]
@KaterinaKrejci231054, thanks for this PR, and @joshuacwnewton, thanks for fixing the CI! 
LGTM now! merging

**Pull request merged**